### DISCUSSION TITLE
Spatial search with content picker

### DISCUSSION THREAD

**Discussion by stevetayloruk** [OrchardCMS/OrchardCore#11866] [Q&A]
Is there a way to use the lucene spatial search with a content picker field?

For example, 

A list of `Events` with a `Store` content picker field.  The geopoint is attached to the `Store` content type.  I want to be able to do a spatial search on events but use the attached `Store`'s geolocation.  (aka a join).

Thanks

**Comment by Skrypt** [comment]
You could do a Spatial ElasticSearch Query on top of the "Store" content type. It will return the proper "Stores" that are within the range that you are looking for. Then, you need to do a second ElasticSearch Query that will find "Events" that are associated with these "Stores".

Not sure if this can be done in a single ElasticSearch Query. I would need to try. That would be a "nested boolean query".

**Reply by stevetayloruk** [reply]
Thanks for the heads-up.  I'll give it a go and report back. 

Thanks
